code/modules/power/tesla/coil.dm

```diff
@@ -90,8 +87,8 @@
 
 /obj/machinery/power/energy_accumulator/tesla_coil/process(seconds_per_tick)
 	. = ..()
-	zap_sound_volume = min(energy_to_joules(stored_energy)/200000, 100)
-	zap_sound_range = min(energy_to_joules(stored_energy)/4000000, 10)
+	zap_sound_volume = min(processed_energy / 8000, 100)
+	zap_sound_range = min(processed_energy / 1.6e5, 10)
```

Comment:
A comment explaining what `8000` & `1.6e5` are used here would be helpful

Reply:
Watts per volume and watts per sound range.

Reply:
I was wrong. It's actually joules.

Reply:
I made it power based and used prefixed units and comments.